projects/schematics/src/shared/utils/lib-utils.ts

```diff
@@ -414,11 +425,16 @@ function addLibraryAssets(assetsConfig: AssetsConfig): Rule {
   };
 }
 
-export function addLibraryStyles(stylingConfig: StylingConfig): Rule {
+export function addLibraryStyles(
+  stylingConfig: StylingConfig,
+  options: LibraryOptions
+): Rule {
   return (tree: Tree, context: SchematicContext) => {
-    const libraryScssPath = `${getSourceRoot(tree)}/styles/spartacus/${
-      stylingConfig.scssFileName
-    }`;
+    const defaultProject = getDefaultProjectNameFromWorkspace(tree);
+    const project = options.project || defaultProject;
+    const libraryScssPath = `${getSourceRoot(tree, {
+      project: project,
+    })}/styles/spartacus/${stylingConfig.scssFileName}`;
     if (tree.exists(libraryScssPath)) {
       context.logger.info(
         `Skipping the creation of '${libraryScssPath}', as it already exists.`
```

Comment:
@psmul did we discuss about removing this check and just appending the content if the scss file already exists?

Reply:
@znikola Yes, exactly. 
Why? We will have situations, where we want to apply 2 sub-features with a single command from one main feature-lib. Example? Product feature-lib, which is just a wrapper for 2 smaller libs that can be applied by one schematics. For now, this causes to apply one scss file but skipping any inclusion of the next ones. This case is rare but might happen more in the future.

Reply:
Looks like topic for another PR and also related to the vision that we would like that each subfeature would have it's own styles, not single styles for the whole library (which is the case at the moment)